build_catalog.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+num_commits=$(git rev-list $(git rev-list --max-parents=0 HEAD)..HEAD --count)
+current_commit=$(git rev-parse --short=7 HEAD)
+version="0.1.$num_commits-$current_commit"
```

Comment:
we can get this from the boilerplate vars: https://github.com/app-sre/deployment-validation-operator/blob/ea2120e886208c1a24a13171d1f8ed10320f5224/boilerplate/openshift/golang-osd-operator/standard.mk#L22-L24

Reply:
Will do this once things are working and can integrate with makefile/boilerplate.  Too many variables atm

Reply:
fair enough